org.eclipse.xtext.testlanguages.ide/src/org/eclipse/xtext/testlanguages/fileAware/ide/quickfix/FileAwareTestLanguageQuickFixProvider.java

```diff
@@ -0,0 +1,28 @@
+package org.eclipse.xtext.testlanguages.fileAware.ide.quickfix;
+
+import org.eclipse.xtext.ide.editor.quickfix.AbstractDeclarativeIdeQuickfixProvider;
+import org.eclipse.xtext.ide.editor.quickfix.DiagnosticResolutionAcceptor;
+import org.eclipse.xtext.ide.editor.quickfix.QuickFix;
+import org.eclipse.xtext.testlanguages.fileAware.fileAware.Element;
+import org.eclipse.xtext.testlanguages.fileAware.validation.FileAwareTestLanguageValidator;
+import org.eclipse.xtext.xbase.lib.StringExtensions;
+
+public class FileAwareTestLanguageQuickFixProvider extends AbstractDeclarativeIdeQuickfixProvider {
+
+	public static class IssueCodes {
+		private IssueCodes() {
+			// utility
+		}
+
+		public static final String TESTLABEL = "Change element name to first upper";
+	}
+
+	@QuickFix(FileAwareTestLanguageValidator.TESTISSUECODE)
+	public void removeUnusedVariable(final DiagnosticResolutionAcceptor acceptor) {
```

Comment:
remove finals

Reply:
@cdietrich see my other comment: If I move this into the testlang, this class will be removed. I'll check if there are any finals left, thanks.